addons/fetchmail/models/fetchmail.py

```diff
@@ -169,14 +169,18 @@ def fetch_mail(self):
                         try:
                             res_id = MailThread.with_context(**additionnal_context).message_process(server.object_id.model, data[0][1], save_original=server.original, strip_attachments=(not server.attach))
                         except Exception:
-                            _logger.info('Failed to process mail from %s server %s.', server.server_type, server.name, exc_info=True)
-                            failed += 1
-                        imap_server.store(num, '+FLAGS', '\\Seen')
+                            _logger.warning('Failed to process mail from %s server %s.', server.server_type, server.name, exc_info=True)
                         self._cr.commit()
-                        count += 1
+                        if res_id:
+                            # imap_server.store(num, '+FLAGS', '\\Seen')
+                            imap_server.store(num, '+FLAGS', '\\Seen \\Flagged')
```

Comment:
You sure about that?

According to [RFC](https://tools.ietf.org/html/rfc3501#section-2.3.2)

> Message is "flagged" for urgent/special attention

so I think it's correct, but how does it appear on webmail clients?

Reply:
Hi, it's just a test. It seems that our emails are being read by someone else than this main process. We've change mail password and still happening, this is just a test to see if this is the method reading all the emails